### ISSUE TITLE
How to distinguish training vs validation loss from tensorboard

### ISSUE THREAD

**Issue by cute-tiger** [tensorflow/models#8326]
When use the" tensorflow object detection api "to training a detection model, and use the recommended directory structure to run tensorboard,command line is tensorboard --logdir=${MODEL_DIR}.

it's output is this image:it have many loss function. Are these loss from training or validation process?  If these are from training process only, how to check validation loss using tensorboard?
<img width="1056" alt="Screen Shot 2020-03-08 at 9 30 40 PM" src="https://user-images.githubusercontent.com/51244825/77436715-33e1c480-6dba-11ea-9116-4dbe80ee090c.png">

**Comment by divyanshusharma1709** [comment]
If you're using the latest TensorFlow Object Detection API (using model_main.py and not train.py), then the Validation Loss can be found under 'loss' and the Training Loss is under 'loss_1' or 'loss_2'.

![Losses](https://user-images.githubusercontent.com/31464367/77502753-bc776800-6e81-11ea-96eb-4ff7afd105d2.png)

**Comment by tispratik** [comment]
Do we have to pass a special validation flag or a file for it to separate out the cross validation data? In our case, even though the validation loss is low, we don't see good generalization.

![Screen Shot 2020-04-15 at 7 47 03 PM](https://user-images.githubusercontent.com/80631/79409169-0d92ed00-7f52-11ea-9122-18c1d95e4c1d.png)
![Screen Shot 2020-04-15 at 7 47 09 PM](https://user-images.githubusercontent.com/80631/79409172-0e2b8380-7f52-11ea-9f1e-30010651fb27.png)

**Comment by ojasvisancheti** [comment]
@divyanshusharma1709 In my case only the loss tag is present, no Loss1 or Loss2, Is i am missing something?
please help
![image](https://user-images.githubusercontent.com/35330613/121905166-c359f700-cd21-11eb-9384-665a811942ab.png)

**Comment by markusbink** [comment]
I'm having the same issue (using model_main.py) that i have multiple losses (i.e. classification loss, localization loss, total loss etc.) but no two curves as @divyanshusharma1709.
